File: Lecture_1_Intro_C++/Java_to_C++/coord.c
#include "coord.h"
#include <iostream>

void Coord::Initialize(double xInit, double yInit) {
	xCoord = xInit;
	yCoord = yInit;
}

void Coord::Reflect() {
	double temp = xCoord;
	xCoord = yCoord;
	yCoord = temp;
}

void Coord::Print() {
	std::cout << xCoord << ", " << yCoord << std::endl;
}

File: Lecture_1_Intro_C++/Coord.java
public class Coord
{

	private double xCoord, yCoord;

	public static void main(String [] args)
	{
		Coord c = new Coord();

		c.Initialize(10.0,4.2);

		c.Print();

		c.Reflect();

		c.Print();
	}

	public void Initialize(double xInit, double yInit)
	{
		xCoord = xInit;
		yCoord = yInit;
	}
	
	public void Reflect()
	{
		double temp = xCoord;
		xCoord = yCoord;
		yCoord = temp;
	}

	public void Print()
	{
		System.out.println(xCoord+", "+yCoord);
	}
}